makeservices/easywp

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+php_defines() {
+    sed -i "s/$1/$2/" "$WPCONFIG"
+}
+
+WEBROOT=$HOME/public_html
+WPCONFIG=wp-config.php
+USER=$(whoami)
+
+# Create symlink to webroot
+makehttp
+cd "$WEBROOT"
+rm -rf *
```

Comment:
You might want a more complicated operation here, as this will leave e.g. `.htaccess` lying around.

Reply:
Yeah... to be honest I'm not sure we ever want to run `rm -rf *` in a script like this.

I totally get why it's here, but I think it's likely to accidentally cause a lot of damage. I could even imagine staff running the script for testing without realizing it would delete their entire public_html/*.

Reply:
I agree, at least make a copy or do a `mv` instead.

Reply:
Yeah. If the user has any files in their public_html before the script is called I'll make a backup in their home folder. 

The wildcard character can match hidden files too. I just need to throw a `shopt -s dotglob` at the beginning of the script.

Reply:
Yeah `mv` would be good. Was going to say earlier that we want to create some sort of backup for the user.